enhancements/cloud-integration/azure/azure-workload-identity.md

```diff
@@ -0,0 +1,401 @@
+---
+title: azure-workload-identity
+authors:
+  - abutcher
+reviewers: # Include a comment about what domain expertise a reviewer is expected to bring and what area of the enhancement you expect them to focus on. For example: - "@networkguru, for networking aspects, please look at IP bootstrapping aspect"
+  - TBD
+approvers:
+  - TBD
+api-approvers: # In case of new or modified APIs or API extensions (CRDs, aggregated apiservers, webhooks, finalizers). If there is no API change, use "None"
+  - TBD
+creation-date: yyyy-mm-dd
+last-updated: yyyy-mm-dd
+tracking-link:
+  - https://issues.redhat.com/browse/CCO-187
+see-also:
+  - "enhancements/cloud-integration/aws/aws-pod-identity.md"
+replaces:
+  - ""
+superseded-by:
+  - ""
+---
+
+# Azure Workload Identity
+
+## Summary
+
+Core OpenShift operators (e.g. ingress, image-registry, machine-api) use long-lived credentials to access Azure API services today. This enhancement proposes an implementation by which OpenShift operators would utilize short-lived, [bound service account tokens](https://docs.openshift.com/container-platform/4.11/authentication/bound-service-account-tokens.html) signed by OpenShift that can be
```

Comment:
Where will the comprehensive list of affected operators be enumerated?

Reply:
The operators are listed in https://issues.redhat.com/browse/CCO-235. I'll add them to this doc.